tools/isomorphic-ws.js

```diff
@@ -0,0 +1,16 @@
+var ws = null;
+
+if (typeof WebSocket !== 'undefined') {
```

Comment:
Isomorphic lib should automatically switch to ws in case of node.js and to global.WebSocket for browsers.